File: SDKs/Aspose.Words-Cloud-SDK-for-Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/HyperlinksResponse.java
package com.aspose.words.model;

import com.aspose.words.model.Hyperlinks;
public class HyperlinksResponse {
  private Hyperlinks Hyperlinks = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getHyperlinks
	 * Gets Hyperlinks
	 * @return Hyperlinks
	 */
  public Hyperlinks getHyperlinks() {
    return Hyperlinks;
  }

	/**
	 * setHyperlinks
	 * Sets Hyperlinks
	 * @param Hyperlinks Hyperlinks
	 */
  public void setHyperlinks(Hyperlinks Hyperlinks) {
    this.Hyperlinks = Hyperlinks;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class HyperlinksResponse {\n");
    sb.append("  Hyperlinks: ").append(Hyperlinks).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/Aspose.Words.Cloud.Sdk/Model/HyperlinksResponse.cs
namespace Aspose.Words.Cloud.Sdk.Model 
{
  using System;
  using System.Text;
  using System.Collections;
  using System.Collections.Generic;
  using System.Runtime.Serialization;
  using Newtonsoft.Json;
  using Newtonsoft.Json.Converters;

  /// <summary>
  /// 
  /// </summary>  
  public class HyperlinksResponse : SaaSposeResponse 
  {                       
        /// <summary>
        /// Gets or sets Hyperlinks
        /// </summary>  
        public Hyperlinks Hyperlinks { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class HyperlinksResponse {\n");
          sb.Append("  Hyperlinks: ").Append(this.Hyperlinks).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
